### PR TITLE
Make the AOT class name configurable

### PR DESCRIPTION
**Pull request by andreaTP**
Follow up from #494

### CODE REVIEW

File: aot/src/main/java/com/dylibso/chicory/aot/AotMachine.java [hunk 1]
```diff
@@ -306,7 +307,15 @@ public final class AotMachine implements Machine {
                     .shared(OpCode.F64_TRUNC, OpcodeImpl.class)
                     .build();
 
+    public static Function<Instance, Machine> withClassName(String className) {
```
Comment (electrum) [inline aot/src/main/java/com/dylibso/chicory/aot/AotMachine.java:310]: The reason to allow setting the class name is for actual ahead-of-time purposes. Users of `AotMachine` today will never need to set this, so I don't think we need this factory.

**Review by electrum — APPROVED** [review]

File: aot/src/main/java/com/dylibso/chicory/aot/AotMachine.java (hunk 1 above)
Comment (andreaTP) [reply aot/src/main/java/com/dylibso/chicory/aot/AotMachine.java:310]: It was just appealing to add it ... if you have a strong opinion, I have no problems removing it as it's just cosmetics :-)

Comment (electrum) [reply aot/src/main/java/com/dylibso/chicory/aot/AotMachine.java:310]: I just don't think it's useful as it won't be used outside of tests. It seems to encourage users to use it, even though it doesn't do anything.

Comment (andreaTP) [reply aot/src/main/java/com/dylibso/chicory/aot/AotMachine.java]: ok, removed :+1:

**Pull request merged**